### PR TITLE
Removed_alpha_beta_ga

### LINKED ISSUE #127782: Remove alpha/beta/ga comments in kube_features.go
We already have detailed version transition information in https://github.com/kubernetes/kubernetes/blob/master/pkg/features/versioned_kube_features.go and the comments in [kube_features.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go) are unnecessary and could drift from the ones documented in code. The versions in `versioned_kube_features.go` are programmatically verified thanks to a utility by @liggitt: https://github.com/kubernetes/kubernetes/pull/127200#discussion_r1768722135 and should be used as the source of truth compared to the comments.

Eg: The alpha and beta lines should be removed.

```
	// owner: @bswartz
	// alpha: v1.18
	// beta: v1.24
	//
	// Enables usage of any object for volume data source in PVCs
	AnyVolumeDataSource featuregate.Feature = "AnyVolumeDataSource"
```

cc @jpbetz @liggitt @aaron-prindle 
/sig architecture

### PR DESCRIPTION
**Pull request by AmarNathChary**
This PR Fixes #127782
Removed Alpha, Beta, Ga

### CODE REVIEW

**Comment by carlory** [comment]
Please update the feature's template in this file too.

**Review by 0x1010110 — APPROVED** [review]

**Comment by Jefftree** [comment]
We should also remove these comments in the kube_features for [apiserver](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/features/kube_features.go), [apiextensions-apiserver](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go), and [controller-manager](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/controller-manager/pkg/features/kube_features.go).

/cc @jpbetz @liggitt 

Does this look like a reasonable change? The version information in `versioned_kube_features.go` is more accurate and duplicating it gives more opportunity for drift.

**Comment by AmarNathChary** [comment]
> We should also remove these comments in the kube_features for [apiserver](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/features/kube_features.go), [apiextensions-apiserver](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go), and [controller-manager](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/controller-manager/pkg/features/kube_features.go).

Would you like me to remove comments in this area too?

**Comment by Jefftree** [comment]
Yes that would be great, and also the template on L30 in kube_features.go

**Comment by AmarNathChary** [comment]
Got it! we make those changes.

File: pkg/features/kube_features.go [hunk 1]
```diff
@@ -806,109 +643,86 @@ const (
...
 	// owner: @ksubrmnn
-	// alpha: v1.14
 	//
 	// Allows kube-proxy to create DSR loadbalancers for Windows
 	WinDSR featuregate.Feature = "WinDSR"
 
 	// owner: @ksubrmnn
-	// alpha: v1.14
-	// beta: v1.20
+
```
Comment (carlory) [inline pkg/features/kube_features.go:683]: it is better to remove this blank line.

**Comment by AmarNathChary** [comment]
@Jefftree done those changes PTAL!

**Comment by liggitt** [comment]
I think the reference to the issue (https://github.com/kubernetes/kubernetes/issues/127782) is probably sufficient ... that has full context

**Comment by Jefftree** [comment]
/lgtm
/retest
fyi unit test flake is fixed https://github.com/kubernetes/kubernetes/pull/128085

**Pull request merged**